Game/Source/GDKTestGyms/BenchmarkGymGameMode.cpp

```diff
@@ -273,19 +329,16 @@ void ABenchmarkGymGameMode::SpawnNPC(const FVector& SpawnLocation)
 		return;
 	}
 
-	const float RandomSpawnOffset = 600.0f;
-	FVector RandomOffset = RNG.GetUnitVector()*RandomSpawnOffset;
-	if (RandomOffset.Z < 0.0f)
-	{
-		RandomOffset.Z = -RandomOffset.Z;
-	}
-	FVector RandomSpawn = SpawnLocation + RandomOffset;
+	FVector FixedSpawnLocation = SpawnLocation*0.9f + BlackboardValues.TargetAValue*0.1f; // Move slightly away from the spawn location
```

Comment:
Why the change?

Reply:
Just so this logic is also deterministic, I doubt the random offset would cause many problems I suppose it could have 2 players run into each other and get stuck

Reply:
Oh, because the movement goes -
communal spawn location > point a > point b > point a (etc.)
Perhaps we should spawn them directly at point a?